source/developer_manual/design/dict_protocol.rst

```diff
@@ -260,6 +260,7 @@ dict client since v2.2.24.
    C: D<transaction id>
 
 .. deprecated:: v2.2.24
+.. versionremoved:: v2.4.0;v3.0.0
 
 The async commit is currently used by dovecot's dict client even though it is
```

Comment:
I don't think we need this paragraph?

Reply:
Removed. I also moved the deprecated & versionremoved to the beginning of the section, should be clearer there than at the end.